tests/integration-tests/conftest.py

```diff
@@ -815,7 +816,7 @@ def create_role(trusted_service, policies=()):
         time.sleep(60)
         logging.info(f"Iam role is ready: {iam_role_name}")
         roles.append({"role_name": iam_role_name, "policies": policies})
-        return iam_role_name
+        return iam_role_name, iam_role_arn
```

Comment:
the uses of `role_factory` will need to be changed to handle the new return type. The only uses I see outside of the multi-user tests are those in test_iam.py:
* https://github.com/aws/aws-parallelcluster/blob/b542d594f3aa9c8980dc1c92ff78df33f7d3bcd0/tests/integration-tests/tests/iam/test_iam.py#L37
* https://github.com/aws/aws-parallelcluster/blob/b542d594f3aa9c8980dc1c92ff78df33f7d3bcd0/tests/integration-tests/tests/iam/test_iam.py#L38
* https://github.com/aws/aws-parallelcluster/blob/b542d594f3aa9c8980dc1c92ff78df33f7d3bcd0/tests/integration-tests/tests/iam/test_iam.py#L60